src/referencePackageSourceGenerator/GenerateProjects/PackageProjectTemplate.xml

```diff
@@ -3,6 +3,8 @@
 
   <PropertyGroup>
     <TargetFrameworks>#TargetFrameworks#</TargetFrameworks>
+	<LangVersion>latest</LangVersion>
```

Comment:
Rethinking about the patterns used in this repo, I think this may be better defined in the common props - https://github.com/dotnet/source-build-reference-packages/blob/main/src/referencePackages/Directory.Build.props.  This gives us more flexibility without having to edit/regen all projects.  The projects should just contain specializations.